src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -1001,6 +1010,12 @@ grpc_error_handle Chttp2ServerAddPort(Server* server, const char* addr,
         }
       }
     }
+    if (resolved_or.ok()) {
```

Comment:
If it's not OK, I think it would have returned already.

Reply:
The code path that populates `resolved` may not be ran. In that case `resolved` is default constructed and has no OK status.

Reply:
I see it now, thanks.